test/unit/testUtil.js

```diff
@@ -67,14 +68,14 @@ define('testUtil', [
     /**
      * Wait for something to happen in the DOM underneath `documentElement`
      * @param {Object} args with keys
-     *      {object}   config           configuration for the MutationObserver
-     *      {string}   documentElement  DOM element to watch
-     *      {function} domStateFunction function that returns true when the watched event happens
-     *      {function} executeFirst     a function to execute after setting up the observer
-     * @param {string} selector to identify the element being watched for
+     * @param {Object} args.config configuration for the MutationObserver
+     * @param {string} args.documentElement  DOM element to watch
+     * @param {function} args.domStateFunction function that returns true when the
+     * watched event happens
+     * @param {function} args.executeFirst     a function to execute after setting up
+     * the observer
```

Comment:
those aren't separate params - they are keys/values in the `args` object

Reply:
Right, that is how objects are documented in jsdoc, I believe. It is a bit weird.

Reply:
Huh. Weird!

Reply:
Although another way of doing this is to create type definitions with @typedef and object properties described with @property, and then just refer to this type in the @param. Unless these types are used by a typechecker, and they serve primarily for documentation (and IDE support if available), I guess best would be whatever is most readable?